opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/RelevanceQueryBuildTest.java

```diff
@@ -67,7 +70,9 @@ void throws_SemanticCheckException_when_wrong_argument_name() {
     FunctionExpression expr =
         createCall(List.of(FIELD_ARG, QUERY_ARG, namedArgument("wrongArg", "value")));
 
-    assertThrows(SemanticCheckException.class, () -> query.build(expr));
+    SemanticCheckException exception =
+        assertThrows(SemanticCheckException.class, () -> query.build(expr));
+    assertEquals("Parameter wrongArg is invalid for mock_query function.", exception.getMessage());
```

Comment:
I'm not sure you want to tie the test that closely to the code (so that if someone changes a single character in the string, the test fails).  
Maybe you can check that the messages contains "wrongArg" and "mock_query" and leave it at that? 
```
assertThat(exception.getMessages(), containsString("wrongArg"));
assertThat(exception.getMessages(), containsString("mock_query"));
```

Reply:
@acarbonetto that's in line with pre-existing convention in this codebase.